components/list/list.js

```diff
@@ -0,0 +1,60 @@
+// Imports
+import React, { PropTypes } from 'react';
+import { getClassNamesWithMods } from '../_helpers';
+
+/**
+ * General List component. Use when you need to display array of elements
+ */
+function List({ items, hideBullets, mods = [], orientation }) {
+  mods.push(`orientation_${orientation}`);
+
+  if (hideBullets) {
+    mods.push("no-bullets");
+  }
+
+  const className = getClassNamesWithMods('ui-list', mods);
+
+  const itemsBlock = items.map((item, index) => (
+    <li key={index}>
+      <span className="ui-list__content">{item}</span>
+    </li>
+  ));
+
+  return (
+    <ul className={className}>
+      {itemsBlock}
+    </ul>
+  );
+}
+
+List.defaultProps = {
+  hideBullets: false,
+  orientation: 'vertical',
```

Comment:
**align** is better word maybe, orientation is confusing with orientation:portrait, orientation:landscape